myst_nbs/diagnostics_and_criticism/model_averaging.myst.md

```diff
@@ -211,7 +220,7 @@ az.plot_forest(traces, figsize=(10, 5));
 
 +++ {"papermill": {"duration": 0.052958, "end_time": "2020-11-29T12:14:55.196722", "exception": false, "start_time": "2020-11-29T12:14:55.143764", "status": "completed"}, "tags": []}
 
-Another option is to plot several traces in a same plot is to use `densityplot`. This plot is somehow similar to a forestplot, but we get truncated KDE plots (by default 95% credible intervals) grouped by variable names together with a point estimate (by default the mean).
+Another option is to plot several traces in a same plot is to use `densityplot`. This plot is somehow similar to a forestplot, but we get truncated KDE (kernel density estimation) plots (by default 95% credible intervals) grouped by variable names together with a point estimate (by default the mean).
```

Comment:
`densityplot` was the old function that generated these kind of plots below. I would now use `plot_density` so it matches the code.

Reply:
Done.